src/View/ViewableData.php

```diff
@@ -668,17 +679,6 @@ public function getViewerTemplates($suffix = '')
         return SSViewer::get_templates_by_class(static::class, $suffix, self::class);
     }
 
-    /**
-     * When rendering some objects it is necessary to iterate over the object being rendered, to do this, you need
-     * access to itself.
-     *
-     * @return ViewableData
-     */
-    public function Me()
-    {
-        return $this;
-    }
```

Comment:
There's no reason to ever call this from PHP code since you already have the object itself at that point.

This has been moved into the templating layer. Without this change, using `<% loop $Me %>` with an `ArrayIterator` or `array` in scope will result in trying to call a `Me()` method on it - which obviously throws errors.

Reply:
Update now that this PR uses `ArrayList` instead of `ArrayIterator`:
I still think it's worth doing this - if you want me to split it out into a separate issue and PR I will but I'm leaving it here for now.